JavaBasicProgram/src/Encapsulationexample/RectangleDemo.java

```diff
@@ -0,0 +1,12 @@
+package Encapsulationexample;
+public class RectangleDemo {
+        public static void main(String[] args)
+        {
+            RectangleArea t=new RectangleArea();
```

Comment:
provide meaningful variable name, should not use single or double character word and meaningless word